cpp/velox/substrait/SubstraitParser.cc

```diff
@@ -391,23 +391,13 @@ std::unordered_map<std::string, std::string> SubstraitParser::substraitVeloxFunc
     {"named_struct", "row_constructor"},
     {"bit_or", "bitwise_or_agg"},
     {"bit_and", "bitwise_and_agg"},
-    {"bitwise_and", "spark_bitwise_and"},
-    {"bitwise_not", "spark_bitwise_not"},
-    {"bitwise_or", "spark_bitwise_or"},
-    {"bitwise_xor", "spark_bitwise_xor"},
-    // TODO: the below registry for rand functions can be removed
-    // after presto function registry is removed.
-    {"rand", "spark_rand"},
     {"murmur3hash", "hash_with_seed"},
     {"xxhash64", "xxhash64_with_seed"},
     {"modulus", "remainder"},
     {"date_format", "format_datetime"},
     {"collect_set", "set_agg"},
-    {"forall", "all_match"},
-    {"exists", "any_match"},
     {"negative", "unaryminus"},
-    {"get_array_item", "get"},
-    {"arrays_zip", "zip"}};
```

Comment:
Have they been registered in Velox sparksql already?

Reply:
Yes, see https://github.com/facebookincubator/velox/pull/10298/files.